paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/DefaultEventReporter.kt

```diff
@@ -464,6 +473,21 @@ internal class DefaultEventReporter @Inject internal constructor(
         }
     }
 
+    private fun fireAnalyticsEvent(event: AnalyticEvent) {
+        CoroutineScope(workContext).launch {
+            analyticEventCallbackProvider.get()?.run {
+                try {
+                    onEvent(event)
+                } catch (@Suppress("TooGenericExceptionCaught") e: Throwable) {
+                    logger.logWarningWithoutPii(
+                        "AnalyticEventCallback.onEvent() failed\n" +
+                            e.stackTraceToString()
```

Comment:
I don't think we want to log the stacktrace to the user facing logger here. We can't be sure it doesn't have Pii.

You could log the stacktrace to our internal logger however, but if we do, we should do both.

I also think it would be helpful to print the event in the warning.